tests/test_notifications.py

```diff
@@ -53,7 +53,8 @@ def test_can_read_children(self):
         sub_component.save()
         sub_component2 = factories.NodeFactory(parent=node)
 
-        has_permission_on_child_node = node.can_read_children(non_admin_user)
+        has_permission_on_child_node =\
```

Comment:
Line continuations are icky. Is there any reason this couldn't be refactored as the following?

```python
assert_true(
    node.has_permission_on_children(non_admin_user, 'read')
)
```